contrib/systemd/etcd2-backup/README.md

```diff
@@ -0,0 +1,249 @@
+# etcd2-backup
+
+Remote backup and multi-node restore services for etcd2 clusters.
+
+## Terminology
+
+**founding member** : The node which is the first member of the new recovered cluster. It is this node's rclone backup data (only) that will be used to restore the cluster. The rest of the nodes will join the cluster with no data, and simply catch up with the **founding member**.
```

Comment:
At the bottom of the document, this caveat is discussed. I'll include the bit about copying the data dir to all the nodes in the case of a large data set.